lib/diagnostic.js

```diff
@@ -6,31 +6,57 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function normalizeWords(paragraph) {
+  let words = paragraph.split(/\s+/);
+  let normalizeWords = [];
+  for (let i = 0; i < words.length; i++) {
+    let word = words[i].replace(/\W+/, '');
+    normalizeWords[i] = word.toUpperCase();
+  }
+
+  return normalizeWords;
 };
 
+console.log(normalizeWords('hjgj h jhg '));
+
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+
+const uniqueWords = function uniqueWords(paragraph) {
+  let uniqueWordsAsKeys = {};
+  let normalizeWords = normalizeWords(paragraph);
```

Comment:
You generally shouldn't give variables the same names as functions.  Here, you're declaring a variable called normalizeWords when there is a function with the same name already.  If you were to try to call normalizeWords again in the same function, it will cause an error.